doc/api/api_changes.rst

```diff
@@ -1,27 +1,150 @@
 :ref:`whats-new`.
 
 
-Changes in 2.0.0
-================
+API Changes in 2.0.0
+====================
 
-Color of Axes
--------------
+Deprecation and removal
+-----------------------
 
+Color of Axes
+~~~~~~~~~~~~~
 The ``axisbg`` and ``axis_bgcolor`` properties on ``Axes`` have been
 deprecated in favor of ``facecolor``.
 
 
+GTK and GDK backends deprecated
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The GDK and GTK backends have been deprecated. These obsolete backends
+allow figures to be rendered via the GDK API to files and GTK2 figures.
+They are untested and known to be broken, and their use has been
+discouraged for some time.  Instead, use the `GTKAgg` and `GTKCairo`
+backends for rendering to GTK2 windows.
+
+WX backend deprecated
+~~~~~~~~~~~~~~~~~~~~~
+The WX backend has been deprecated.  It is untested, and its
+use has been discouraged for some time. Instead, use the `WXAgg`
+backend for rendering figures to WX windows.
+
+CocoaAgg backend removed
+~~~~~~~~~~~~~~~~~~~~~~~~
+The deprecated and not fully functional CocoaAgg backend has been removed.
+
+
+`Artist.update` has return value
+--------------------------------
+
+The methods `matplotlib.artist.Artist.set`,
+`matplotlib.Artist.update`, and the function `matplotlib.artist.setp`
+now use a common codepath to look up how to update the given artist
+properties (either using the setter methods or an attribute/property).
+
+The behavior of `matplotlib.Artist.update` is slightly changed to
+return a list of the values returned from the setter methods to avoid
+changing the API of `matplotlib.Artist.set` and
+`matplotlib.artist.setp`.
+
+The keys passed into `matplotlib.Artist.update` are now converted to
+lower case before being processed, to match the behavior of
+`matplotlib.Artist.set` and `matplotlib.artist.setp`.  This should not
+break any user code because there are no set methods with capitals in
+their names, but this puts a constraint on naming properties in the future.
+
+
+`Legend` initializers gain edgecolor and facecolor kwargs
+---------------------------------------------------------
+
+The :class:`~matplotlib.legend.Legend` background patch (or 'frame')
+can have its ``edgecolor`` and ``facecolor`` determined by the
+corresponding keyword arguments to the :class:`matplotlib.legend.Legend`
+initializer, or to any of the methods or functions that call that
+initializer.  If left to their default values of `None`, their values
+will be taken from ``matplotlib.rcParams``.  The previously-existing
+``framealpha`` kwarg still controls the alpha transparency of the
+patch.
+
+
+Qualitative colormaps
+---------------------
+
+Colorbrewer's qualitative/discrete colormaps ("Accent", "Dark2", "Paired",
+"Pastel1", "Pastel2", "Set1", "Set2", "Set3") are now implemented as
+``ListedColormap`` instead of ``LinearSegmentedColormap``.
+
+To use these for images where categories are specified as integers, for
+instance, use::
+
+    plt.imshow(x, cmap='Dark2', norm=colors.NoNorm())
```

Comment:
Yes, I'll have a look.

On Tue, Sep 6, 2016 at 1:39 AM, Thomas A Caswell <notifications@github.com>
wrote:

> In doc/api/api_changes.rst
> <https://github.com/matplotlib/matplotlib/pull/6994#discussion_r77559531>:
>
> > +will be taken from ``matplotlib.rcParams``.  The previously-existing
> > +``framealpha`` kwarg still controls the alpha transparency of the
> > +patch.
> > +
> > +
> > +Qualitative colormaps
> > +---------------------
> > +
> > +Colorbrewer's qualitative/discrete colormaps ("Accent", "Dark2", "Paired",
> > +"Pastel1", "Pastel2", "Set1", "Set2", "Set3") are now implemented as
> > +``ListedColormap`` instead of ``LinearSegmentedColormap``.
> > +
> > +To use these for images where categories are specified as integers, for
> > +instance, use::
> > +
> > +    plt.imshow(x, cmap='Dark2', norm=colors.NoNorm())
>
> Could you add NoNorm to this page: http://matplotlib.org/users/
> colormapnorms.html
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/6994/files/6d97f09cd95ebb90d1aadb67238ca7eb121fa7c7#r77559531>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAMm9YuAaU42p7_fG3lsXM_5OPteN_3Bks5qnKg3gaJpZM4JvC49>
> .
>